### ISSUE TITLE
Add Github Workflow that Deploys Airflow Sandbox Data Pipeline Container to ECS on Pushes to Main Branch

### ISSUE THREAD

**Issue by njfritter** [Qbizinc/sandbox-data-pipeline#22]
## Description

The Workflow in #20 was successfully launched onto AWS ECS using the test branch `workflows`. Now that the POC has been complete, we can move forward and add a workflow that deploys the Airflow Sandbox Data Pipeline container onto AWS ECS whenever there are pushes to the `main` branch (i.e. specifically when a PR is approved and merged into `main`).

## Definition of Done

A workflow file that deploys the Airflow Sandbox Data Pipeline container onto AWS ECS whenever there are pushes to the `main` branch has been created and added to `main`.

**Comment by njfritter** [comment]
Currently waiting on #21 to be merged.
